wntr/network/io.py

```diff
@@ -350,7 +350,7 @@ def from_gis(gis_data, append=None):
     if isinstance(gis_data, dict):
         gis_data = WaterNetworkGIS(gis_data)
 
-    wn = gis_data._create_wn(append=append)
+    wn = gis_data._create_wn(append=append,index_col=index_col)
```

Comment:
This change causes an error because index_col is not defined in this context. It is also no longer necessary given the earlier suggestions.